src/ostorlab/cli/scan/run/assets/android_apk.py

```diff
@@ -48,10 +50,24 @@ def android_apk(
 
     logger.debug("scanning assets %s", [str(asset) for asset in assets])
     try:
-        runtime.scan(
+        created_scan = runtime.scan(
             title=ctx.obj["title"],
             agent_group_definition=ctx.obj["agent_group_definition"],
             assets=assets,
         )
+
+        if isinstance(ctx.obj["runtime"], runtime_local.LocalRuntime) is True:
```

Comment:
As discussed, update runtime to add a method to perform the action and the other runtimes don't do anything in this case.